pymc3/tests/test_shape_handling.py

```diff
@@ -28,6 +31,8 @@
     to_tuple,
 )
 
+_log = logging.getLogger(__file__)
```

Comment:
Why would we need logging in a test module?  If this is for local debugging, then it should be removed; otherwise, there's no reason to add noise to the output of our tests.

Reply:
I used it just for logging and forgot to remove it before committing. Will do.